Automattic/simplenote-electron
# Windows: update with renewed codesigning cert

### Description

Windows builds in CI were broken today due to an expired code signing cert. This PR updates the cert to the latest-and-greatest.

### To Test

Verify you can successfully install the Windows artifact from this build.  

https://github.com/Automattic/wp-desktop/pull/843
